tools/distrib/pylint_code.sh

```diff
@@ -32,7 +32,7 @@ TEST_DIRS=(
 )
 
 VIRTUALENV=python_pylint_venv
-python3 -m virtualenv $VIRTUALENV
+python3 -m virtualenv $VIRTUALENV -p $(which python3)
```

Comment:
Is the `virtualenv` creating an environment with a different CPython binary?

Reply:
Yup. But no more.